File: hello.c
#include<stdio.h>

int main() {
	int a = 2;
	int b = 2;
	int c = a + b;
	
	printf("C says: Hello, World\n");
	printf("%d + %d = %d\n",a,b,c);
	
	char *listOfUsers[] = {"User1", "User2", "User2"};
	int users = sizeof(listOfUsers) / sizeof(listOfUsers[0]);

	for (int i = 0; i < users; i++) {
		printf("%s", listOfUsers[i]);
		if (i < users - 1) {
			printf(", ");
		}
	}
	printf("\n");
	return 0;
}

File: hello.py
#!/usr/bin/env python3
a = 2
b = 2
c = a + b
print("Python says: Hello, World!")
print("%s + %s = %s" % (a,b,c))
listOfUsers = ["User1", "User2", "User3"];
for i, user in enumerate(listOfUsers):
    if i == len(listOfUsers) - 1:
            print(user)
    else:
            print(user, end=', ')
